packages/@aws-cdk/aws-codebuild/package.json

```diff
@@ -129,10 +129,12 @@
       "docs-public-apis:@aws-cdk/aws-codebuild.Cache.none",
       "docs-public-apis:@aws-cdk/aws-codebuild.LinuxBuildImage.AMAZON_LINUX_2",
       "docs-public-apis:@aws-cdk/aws-codebuild.LinuxBuildImage.AMAZON_LINUX_2_2",
+      "docs-public-apis:@aws-cdk/aws-codebuild.LinuxBuildImage.AMAZON_LINUX_2_3",
```

Comment:
Can you add a comment instead of this exclusion?

```ts
/** The Amazon Linux 2 x86_64 standard image, version `3.0`. */
public static readonly AMAZON_LINUX_2_3 = LinuxBuildImage.codeBuildImage('aws/codebuild/amazonlinux2-x86_64-standard:3.0');
```

Reply:
@skinny85 no problem this has been done.